src/pkg/bundle/deploy.go

```diff
@@ -233,6 +234,69 @@ func handleZarfInitOpts(pkgVars zarfVarData, zarfPkgKind v1alpha1.ZarfPackageKin
 	return zarfInitOpts
 }
 
+// PreDeployValidationTF validates a bundle that has been built from a .tf file before deployment
+// TODO: This function shares (copied) a lot of code from PreDeployValidation(). Work towards consolidating what makes sense.
```

Comment:
I apologize for my sins; I will look at this a bit tomorrow morning.

Reply:
Meh, I'm sure the compiler reduces it to shared code anyway :P